spyder/plugins/debugger/widgets/framesbrowser.py

```diff
@@ -545,7 +545,7 @@ def set_frames(self, stack_dict):
                     self.data[id(item)] = (frame["filename"], frame["lineno"])
             else:
                 item = LineFrameItem(
```

Comment:
```suggestion
                item = LineFrameItem(parent, 0, None, '', 0, '')
```

Let's leave this in a single line because it fits in less than 79 columns.